data/tr2/ship/cfg/TR2X_gameflow_gm.json5

```diff
@@ -0,0 +1,137 @@
+        {
+            "type": "gym",
+            "path": "data/assault.tr2",
+            "music_track": -1,
+            "sequence": [
+                {"type": "loop_game"},
+                {"type": "level_stats"},
+            ],
+        },
+
+        // 1. The Cold War
+        {
+            "path": "data/level1.tr2",
+            "music_track": 33,
+            "sequence": [
+                {"type": "loop_game"},
+                {"type": "play_music", "music_track": 41},
+                {"type": "level_stats"},
+                {"type": "level_complete"},
+            ],
+        },
+
+        // 2. Fool's Gold
+        {
+            "path": "data/level2.tr2",
+            "music_track": 58,
+            "sequence": [
+                {"type": "loop_game"},
+                {"type": "play_music", "music_track": 41},
+                {"type": "level_stats"},
+                {"type": "level_complete"},
+            ],
+        },
+
+        // 3. Furnace of the Gods
+        {
+            "path": "data/level3.tr2",
+            "music_track": 59,
+            "sequence": [
+                {"type": "loop_game"},
+                {"type": "play_music", "music_track": 41},
+                {"type": "level_stats"},
+                {"type": "level_complete"},
+            ],
+        },
+
+        // 4. Kingdom
+        {
+            "path": "data/level4.tr2",
+            "music_track": 31,
+            "sequence": [
+                {"type": "give_item", "object_id": "puzzle_1"},
+                {"type": "loop_game"},
+                {"type": "play_music", "music_track": 52},
+                {"type": "display_picture", "path": "data/credit00_gm.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit01.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit02.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit03.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit04.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit05.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit06.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit07_gm.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "display_picture", "path": "data/credit08.pcx", "display_time": 15, "fade_in_time": 0.5, "fade_out_time": 0.5},
+                {"type": "total_stats", "background_path": "data/end.pcx"},
+                {"type": "level_complete"},
```

Comment:
I think we forgot a
`{"type": "level_stats"},`
here? Upon Kingdom's completion, no level stats are shown before the credits.

Reply:
Thanks, this is how it's configured in the original though. HSH is similar, so it goes straight to credits.

Reply:
Do you think we should change it? It's kinda weird not to have stats.

Also semi related but a different issue idea, it would be nice to give some kind of visual indicator in the final stats about missing secrets causing you to miss out on the bonus level. The only way you know it exists is from Googling or other fans.

Reply:
I'd opt for adding it but in a way that plays the skidoo theme already. A separate effort by all means.

Reply:
@walkawayy would you mind opening the separate tickets if you get a chance? Hopefully then we can draw a line under this PR.